proto/viam/app/data/v1/data.proto

```diff
@@ -2,29 +2,29 @@ syntax = "proto3";
 
 package viam.app.data.v1;
 
+import "google/protobuf/any.proto";
+import "google/protobuf/struct.proto";
 import "google/protobuf/timestamp.proto";
 
 option go_package = "go.viam.com/api/app/data/v1";
 
 // DataService is used to query robot data.
 service DataService {
-  // Query queries metadata based on given filters.
-  rpc Query(viam.app.data.v1.QueryRequest) returns (viam.app.data.v1.QueryResponse);
+  // TabularDataByFilter queries tabular data and metadata based on given filters.
+  rpc TabularDataByFilter(TabularDataByFilterRequest) returns (stream TabularDataByFilterResponse);
+
+  // BinaryData apis are used for both images and file data
```

Comment:
nit: perhaps break these up since I believe they autogenerate documentation?

// BinaryDataByFilter queries binary data and metadata based on given filters.
// BinaryDataByIDs queries binary data and metadata based on given file IDs.

Reply:
Bump here before submitting as well. Thanks!